<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimelineTopics</class>
 <widget class="QWidget" name="TimelineTopics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Total duration:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="totalDurationEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string notr="true">HH:mm:ss</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Topic</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Icon</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Duration</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="KPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="removeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="upButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Move Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KPushButton" name="downButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Move Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
